### ISSUE TITLE
96000 Hz

### ISSUE THREAD

**Issue by legosoff** [nu774/qaac#94]
Hi. Qaac 96000 Hz does not support. When will it be updated?

**Comment by nu774** [comment]
Nope. It's limitation of the encoder (which is developed by Apple)
You can see it with qaac --format.
